examples/angular2/index.html

```diff
@@ -0,0 +1,24 @@
+<!doctype html>
+<html lang="en" data-framework="angular2">
+	<head>
+		<meta charset="utf-8">
+		<title>Angular2 • TodoMVC</title>
+		<link rel="stylesheet" href="node_modules/todomvc-app-css/index.css">
```

Comment:
`<link rel="stylesheet" href="node_modules/todomvc-common/base.css">` should be above here